sdk/nextjs/src/server/requests.ts

```diff
@@ -0,0 +1,33 @@
+import { getSDKKey } from './requestContext'
+
+const getFetchUrl = () =>
+    `https://config-cdn.devcycle.com/config/v1/client/${getSDKKey()}.json`
+
+export const fetchCDNConfig = async (lastModified?: string) => {
+    const response = await fetch(
+        getFetchUrl(),
+        // only store for 30 seconds, or force revalidate
+        { next: { revalidate: 30 } },
+    )
+
+    const lastModifiedHeader = response.headers.get('last-modified')
+
+    const lastModifiedCache = new Date(lastModifiedHeader ?? 0)
+    const lastModifiedClient = new Date(lastModified ?? 0)
+
+    if (
+        lastModifiedHeader &&
+        lastModified &&
+        lastModifiedClient > lastModifiedCache
+    ) {
+        console.log('REFETCHING NO CACHE')
+        return await fetch(
```

Comment:
expire the Next cache when the client tells us it had a config that is newer than the cached one we have here